Add cama

hii600 opened the pull request:
Hi Jannis and Dirk,

Exposure of the input runoff variable has been finished. In order to check, first compile by "**make all**" in _cama-flood_bmi_ directory, go to _test_ directory, and then run "**python test_elbe.py**".

For the test run, the sample input runoff is given in the _sample_runoff_ directory. This is regional runoff data of the same domain as PCR but at the different resolution (1 degree).

I hope this works, but let me know if you find some problem.

Thank you,
Hiroaki

ChippChapp commented:
Hi @hii600 ,

doesn't work for me unfortunately.
If compiling with "make all", the makefile in the mod-folder requires the Mkinclude file in the adm-folder which is unfortunately not there (the entire adm-folder I mean).

Side question:
Did you make the same changes for the actual CaMa-BMI repository?

Cheers, Jannis

hii600 commented:
Hi @ChippChapp ,

thank you for your check. Yes that's correct, because I added it in .gitignore file. So can you copy the same adm/Mkinclude file as in your CaMa-BMI directory?

I didn't make the same changes to the CaMa-BMI repository.

Thanks, Hiroaki

ChippChapp commented:
@hii600 I can confirm that it works if copying the files from the "actual" cama-bmi repos to this test-repos here. Great!

P.S. Great for now but ofc we should align that at one point.

ChippChapp commented:
@hii600 just pushed all other folders required to get this version of CaMa-BMI here working for everyone without copying folders.
Could you maybe check which folders/files are not futher needed for the coupling test case and remove them?

Thanks a lot!

hii600 commented:
Hi @ChippChapp 
Thanks for your suggestion. Now we don't need gosh and map directories (compile.sh is the same as 'make all' command, and map directory is already in test_elbe directory), so I removed them.
Instead of copying adm directory afterwards, I added sample Mkinclude file and adm directory which contains Mkinclude file. By doing so we can get adm directory and then edit it by following the user's computer environment.

Cheers!
